src/alda/error/IOException.java

```diff
@@ -0,0 +1,15 @@
+package alda.error;
+
+public class IOException extends AldaException {
```

Comment:
While IOException does make it very clear that this is a wrapper of IOException, I think it would be better to give it a different name, just so it does not conflict with `java.io.IOException` (and then make it clear via comment's that this is a IOExeption). Maybe a `InputException` or a `ReadException` or a `CommException`?

This would also prevent (slight) confusion in a case where you did not import `java.io.IOException` but imported this one instead.